backend/development.ini.sample

```diff
@@ -439,7 +439,7 @@ website.base_url = %(basic_setup.website_base_url)s
 # EVENT_TYPE   = created | modified | deleted | undeleted
 # CONTENT_TYPE = html-document | file | thread | folder | comment
 # To include all notifications, use web.notifications.excluded = ""
-; web.notifications.excluded = user.created, user.modified, user.deleted, user.undeleted, workspace.modified, workspace.deleted, workspace.undeleted, workspace_member.modified, content.modified
+; web.notifications.excluded = user.*, workspace.modified, workspace.deleted, workspace.undeleted, workspace_member.modified, content.modified, reaction.*
```

Comment:
If we chnage default value we need to add information in chnagelog about that.
Please add comment and information in this issue #4282

Reply:
https://github.com/tracim/tracim/pull/4278#discussion_r596959328

Reply:
I put a comment there 3h ago @PhilippeAccorsi @inkhey